langgraph/pregel/read.py

```diff
@@ -134,9 +140,12 @@ def get_node(self) -> Optional[Runnable[Any, Any]]:
         elif self.bound is DEFAULT_BOUND:
             return RunnableSequence(*writers)
         elif writers:
-            return RunnableSequence(self.bound, *writers)
+            return RunnableSequence(
+                self.bound.bind(**kwargs) if kwargs is not None else self.bound,
+                *writers,
+            )
         else:
-            return self.bound
+            return self.bound.bind(**kwargs) if kwargs is not None else self.bound
```

Comment:
need to do .bind() as RunnableSequence doesn't propagate kwargs to each step (nor should it I think)